packages/datadog-plugin-mocha/src/index.js

```diff
@@ -71,6 +74,9 @@ function createWrapRunTest (tracer, testEnvironmentMetadata, sourceRoot) {
             }
           } catch (error) {
             activeSpan.setTag(TEST_STATUS, 'fail')
+            activeSpan.setTag(ERROR_TYPE, error.constructor ? error.constructor.name : error.name)
```

Comment:
`setTag` supports being called with an error like so: `setTag('error', error)`. Could that be used to simplify this code?

Reply:
I tried just that. It doesn't seem to work, at least for `jest`. I think it is because the error thrown by `jest` on assertion errors does not have a standard shape. This is more verbose but safer

Reply:
so the problem is that for whatever reason, `error instanceof Error` in https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/format.js#L118 is false for an `error` thrown by `jest`. I'll investigate it further.
I can update it here in `mocha` and leave it for `jest`.
WDYT?